src/app/routes/utils/overrideRendererOnTest/index.js

```diff
@@ -0,0 +1,18 @@
+import Url from 'url-parse';
+import isLive from '#lib/utilities/isLive';
+
+export const addOverrideQuery = (path) => {
+  const url = new Url(path);
+  const { query } = url;
+  const searchParams = new URLSearchParams(query);
+  searchParams.set('renderer_env', 'live');
+  const searchString = searchParams.toString();
+  const pathName = path ? path.split('?')[0] : path;
+
+  return `${pathName}?${searchString}`;
+};
+
+const overrideRendererOnTest = (path) =>
```

Comment:
Can we rename this function / file to something like - getRadioDataPath (and abstract away the detail of the override)

Reply:
We have agreed to leave the name as is.